Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -66,6 +68,18 @@ export const projectDispatcher = () => {
     }
   );
 
+  const removeRemoteSkillFromBotProject = useRecoilCallback(
```

Comment:
@zhixzhan  do we need this function?

removeSkillFromBotProject does the same thing right?